database/utils/async-for-each.js

```diff
@@ -1,20 +1,20 @@
 require('dotenv').config();
 
-const { WAITTIME_BETWEEN_CALLS } = process.env;
-const waitBetweenCalls =  Number(WAITTIME_BETWEEN_CALLS);
+const WAITTIME_BETWEEN_CALLS = process.env.WAITTIME_BETWEEN_CALLS || 1;
+const waitBetweenCalls = Number(WAITTIME_BETWEEN_CALLS);
 
 const wait = (seconds) => {
   seconds === 0 ? 1 : seconds;
```

Comment:
Is the goal of line 7 to make sure if someone actually specifies the value `0` for `WAITTIME_BETWEEN_CALLS` in the `.env` file or to just make sure a value of `1` is used for `seconds` if nothing is passed to the function?  Since line 3 deals with the latter case, then the code only seems to be making sure a value of 1 or greater is always passed to the function.

Reply:
```suggestion
const wait = (seconds) => {
  seconds = seconds < 1 ? 1 : seconds;
```
@RandellDawson I assume so, yeah.  I'd go with this, though, since you know seconds is a number and you want it to be >=1.